<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RS232Dialog</class>
 <widget class="QDialog" name="RS232Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>621</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Protocol For communication</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baudrate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Bits</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_stopbits">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parity</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_parity">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_serial">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_port">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Port</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>621</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_update">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Firmware</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
